### PR TITLE
ARCH-1690 - Move actions to repo

### PR DESCRIPTION
**Pull request by danielle-casella-adams**
# Summary of PR changes

- Move did-custom-action-code-change and update-action-version-in-file into this repository.
- Add test workflow for both actions
- Modify the reusable workflows to use a local copy of these two actions.

### CODE REVIEW

File: .github/actions/did-custom-action-code-change/src/main.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  } else {
+    core.info(`\nNo folders had changes in the PR.`);
+  }
+
+  return prFolders;
+}
+
+async function getPRFiles(org, repo, prNumber) {
+  let prFilesSet = new Set();
+  let prFiles = [];
```
Comment (hpractv) [inline .github/actions/did-custom-action-code-change/src/main.js]: This is really nitpicking, but does this need to be declared at this scope?  You're only using this at the `.then`  for enumeration in favor of filling up `prFileSet`.

**Review by hpractv — DISMISSED** [review]
Just one small thing you don't have to fix if you don't want to.

File: .github/actions/did-custom-action-code-change/src/main.js (hunk 1 above)
Comment (danielle-casella-adams) [reply .github/actions/did-custom-action-code-change/src/main.js]: It's not nitpicking.  You're right about the scope and I can change the the `let` for `prFilesSet` to a const.  I was trying different things out and that was a leftover.

**Review by hpractv — APPROVED** [review]
👍🏽

**Pull request merged**